### DISCUSSION TITLE
Add BOINC

### DISCUSSION THREAD

**Discussion by Supermagnum** [swizzin/swizzin#1105] [Feature Requests]
BOINC lets you help cutting-edge science research using your computer. The BOINC app, running on your computer, downloads scientific computing jobs and runs them invisibly in the background. It's easy and safe.

About 30 science projects use BOINC. They investigate diseases, study climate change, discover pulsars, and do many other types of scientific research. 

It is located here:
https://boinc.berkeley.edu/

**Comment by albertusQuelle** [comment]
What should that benefit the project? You can simply install SETI aka BOINC standalone??
